### PR TITLE
[Vmware] Fix worker VM invalid numeric value

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR fixes an issue on VMware when creating worker VMs for volume migration from/to hosts with different ESXi versions. In this case, the minimum ESXi host version is chosen for the worker VM hardware version but it is set in a wrong format ("DD" instead of "vmx-DD")
 
### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [x] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [x] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
Vmware mixed environment with 2 clusters of 2 hosts each. One cluster with 5.5 hosts and other cluster with 6.5 hosts

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan test centos7 vmware-67u3

**Review by shwstppr — APPROVED** [review]

**Review by rhtyd — APPROVED** [review]

**Review by Pearl1594 — APPROVED** [review]
code LGTM

**Comment by rhtyd** [comment]
Let's wait for @andrijapanicsb 's test confirmation, before merging this, thanks.

**Review by rhtyd — APPROVED** [review]

**Comment by rhtyd** [comment]
Discussed with @andrijapanicsb - he's advised to go ahead with the merge and kick smoketests 6.7u3.

**Pull request merged**
